src/CaNNOLeS.jl

```diff
@@ -623,15 +623,34 @@ function SolverCore.solve!(
   return stats
 end
 
-function newton_system(x, r, λ, Fx, rhs, LDLT, ρold, params, method, linsolve)
```

Comment:
Maybe leave a function with the old signature that calls the new one, and mark it as deprecated?

Reply:
Thanks for the comments. This is added.